torch/distributed/distributed_c10d.py

```diff
@@ -278,6 +278,22 @@ def __new__(cls, name: str):
             value = name.lower()
         return value
 
+    @staticmethod
+    def get_default_backend(device) -> str:
```

Comment:
@H-Huang @kwen2501 any concerns with this api?

Reply:
@H-Huang @kwen2501 , could you please review.